scripts/machineconfig/99-sfc-machineconfig.bu

```diff
@@ -0,0 +1,73 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2023 Advanced Micro Devices, Inc.
+
+variant: openshift
+version: OPENSHIFT_VERSION
+metadata:
+  name: 99-sfc-machineconfig
+  labels:
+    machineconfiguration.openshift.io/role: NODE_TYPE
+systemd:
+  units:
+    - name: sfc-replace-kernel-module.service
+      enabled: true
+      contents: |
+        [Unit]
+        Description=Replace in-tree kernel module with oot kernel module
+        Before=network-pre.target
+        Wants=network-pre.target
+        DefaultDependencies=no
+
+        [Service]
+        User=root
+        Type=oneshot
+        TimeoutSec=10
+        ExecStartPre=ls /usr/local/bin
+        ExecStart=/usr/local/bin/sfc-replace-kernel-module.sh
+        PrivateTmp=yes
+        RemainAfterExit=no
+        TimeoutSec=60
+        Environment="ONLOAD_MODULE_IMAGE=ONLOAD_REPO_IMAGE/ONLOAD_REPO_TAG"
```

Comment:
Is this out of date now? ONLOAD_REPO_IMAGE and ONLOAD_REPO_TAG don't exist outside of this file.
Should it be something like:
```suggestion
        Environment="ONLOAD_MODULE_IMAGE=ONLOAD_REPO_URL"
```